tests/framework/pkg/session/session.go

```diff
@@ -53,7 +54,7 @@ func (ts *Session) Cleanup() {
 
 // NewSession returns a `Session` who's cleanup method is registered with this `Session`
 func (ts *Session) NewSession() *Session {
-	sess := NewSession(ts.TestingT)
+	sess := NewSession()
```

Comment:
Same nit as Ugur above, but seems this was by accident 😄

Reply:
I removed TestingT because we could just use logrus to log things within the framework rather than passing the T just to log